hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/protocolPB/ClientNamenodeProtocolServerSideTranslatorPB.java

```diff
@@ -25,22 +25,21 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-import org.apache.hadoop.thirdparty.protobuf.ByteString;
-import org.apache.hadoop.thirdparty.protobuf.ProtocolStringList;
 import org.apache.hadoop.classification.InterfaceAudience;
 import org.apache.hadoop.classification.InterfaceStability;
 import org.apache.hadoop.fs.BatchedRemoteIterator.BatchedEntries;
+import org.apache.hadoop.fs.ContentSummary;
```

Comment:
The sorting is correct, but I would avoid these changes to reduce churn.

Reply:
I thought i was asked to do this previously, but couldn't find the comment so I undid the change.
cc @steveloughran do you remember if you asked me to do this? if so ill revert the change that puts it back